types/accept-language-parser/index.d.ts

```diff
@@ -10,7 +10,7 @@
 export function parse(acceptLanguage?: string): Language[];
 export function pick<T extends string>(
     supportedLanguages: T[],
-    acceptLanguage: string | Language[],
+    acceptLanguage: string | Language[] | null,
```

Comment:
```ts
    acceptLanguage: string | Language[] | null,
```
why this change? Docs (readme) and tests, never expect anything but string here. Source code works with anything that evaluates to false, but this is hardly expect to be advertised in docs, so why 'null' as very specific type in JS?

Reply:
From https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61235#issue-1304026307:

> `pick` accepts any falsy values for its parameters: https://github.com/opentable/accept-language-parser/blob/3a1133a42889482e768f01d0b4e4384bb14aa762/index.js#L34
>
>In general passing a falsy value is probably a bug. However, since `Headers#get` returns `string | null` (https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L8877), it's handy to allow `null` as `acceptedLanguage` so that this pattern is accepted:
>
> ```js
> pick(supportedLanguages, headers.get("accepted-language"));
> ```